file-attribute-caching/build.gradle.kts

```diff
@@ -84,8 +85,12 @@ publishing {
 }
 
 signing {
-    // Signing credentials are stored locally in the user's global gradle.properties file.
-    // See https://docs.gradle.org/current/userguide/signing_plugin.html#sec:signatory_credentials for more information.
+    useInMemoryPgpKeys(
+        signingKeyId,
```

Comment:
https://docs.gradle.org/current/userguide/signing_plugin.html?cmdf=gradle+useinmemorypgpkey#sec:in-memory-keys suggests the key ID isn't needed

Reply:
Oh I see, for pgp subkeys. all good.